monitoring/monitorlib/messagesigning/message_signer.py

```diff
@@ -0,0 +1,117 @@
+import base64
+import time
+from Crypto.PublicKey import RSA
+from Crypto.Signature import pkcs1_15
+from Crypto.Hash import SHA256
+from loguru import logger
+import urllib.parse
+from monitoring.monitorlib.messagesigning.hasher import get_content_digest
+
+
+def get_x_utm_jws_header(cert_url):
+    return '"alg"="{}", "typ"="{}", "kid"="{}", "x5u"="{}"'.format(
```

Comment:
I couldn't track down an official statement from the RFC but is wrapping the dictionary key in double-quotes proper/allowed?
None of the examples I've seen have quoted dictionary keys, and it breaks the http-sfv parsing library I'm using...

Should this line be:
```
return 'alg="{}", typ="{}", kid="{}", x5u="{}"'.format(
```

Reply:
Good catch! Will make that change asap